react/queries/locales.gql

```diff
@@ -3,4 +3,7 @@ query Locales {
     default
     supported
   }
+   currentBinding {
```

Comment:
can you put `currentBinding @context(provider: 'vtex.tenant-graphql')`

Its a standard that we want in the front end but we forget to enforce